python/ray/tune/tests/test_result_grid.py

```diff
@@ -224,6 +224,62 @@ def f(config):
     assert not any(key in representation for key in AUTO_RESULT_KEYS)
 
 
+def test_result_grid_repr(ray_start_2_cpus, tmpdir):
```

Comment:
No longer need to do ray init if we create the result objects directly.
```suggestion
def test_result_grid_repr(tmpdir):
```